lib/db_utils.pm

```diff
@@ -61,18 +66,25 @@ Example of data:
 sub influxdb_push_data {
     my ($url, $db, $org, $token, $data, %args) = @_;
     $args{quiet} //= 1;
+    $args{proceed_on_failure} //= 0;
```

Comment:
well when we speaking about boottime test or some fio test when half of the test value is in collecting perf measurements into DB than I tend to disagree

Reply:
Based on this default all the func. calls should be updated accordingly.
The actual running logic is based on `die` unconditionally, so that default has been kept. 
But we can agree a different default.

Reply:
Default `false` means that,
- the _new boottime management_, via `create_instance`, going in **perf_2** bucket, the related func. calls have been updated to force `true` pass;
- the preexisting _boottime and storage_perf_ routines, going in **perf** bucket, instead, should use that default `false`.